### DISCUSSION TITLE
template variables belong to row，not dashboard

### DISCUSSION THREAD

**Discussion by yuandongjian** [grafana/grafana#37733] [Dashboards, Panels & Visualizations]
In the dashboard，we use row to distinguish the different metrics，and wo use some template variables in the dashboard.
However, template variables are not useful in all rows，we want to be able to make our template variables belong to row.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
